### ISSUE TITLE
All encoders failed to encode

### ISSUE THREAD

**Issue by plusplus99** [rapid7/metasploit-framework#14130]
I am trying to use this exploit on HTB : https://www.rapid7.com/db/modules/exploit/windows/smb/ms08_067_netapi

pretty simple yet metasploit gives me an error : **all encoders failed to encode** 

I have spent hours trying to find a solution but I found nothing :/ 

This is screenshot for the error https://ibb.co/sQ5cCFY

Metasploit version is 6.0.2-dev

I am really frustrated and I am about to give up on my first box.

**Comment by franferrax** [comment]
Hi, sorry for the duplicate. I've just reported the same issue in #14141. I provided further information to help solving this issue (including the 6f153688ffe0b7ba8eae893b8d10c265bb351aab commit spotted by `git bisect`) but I'm afraid I don't have a fix or workaround for it.

**Comment by IdanSaadon** [comment]
Hey!
I had the same issue. After a lot of attempts I got the solution - just change the payload.
you can use **`show payloads`** to see all payloads. I chose the payload *windows/meterpreter/reverse_tcp_allports* cause it's very similar to the original (by name, the original is *windows/meterpreter/reverse_tcp*).
command:
```php
msf6 exploit(windows/smb/ms08_067_netapi) > set payload windows/meterpreter/reverse_tcp_allports
```
Good Luck! :)

**Comment by smcintyre-r7** [comment]
This was fixed when #14177 landed.
